lib/pdk/module/metadata.rb

```diff
@@ -54,12 +54,16 @@ class Metadata
           'operatingsystem'        => 'windows',
           'operatingsystemrelease' => ['2008 R2', '2012 R2', '10'],
         },
+        'Archlinux' => {
+          'operatingsystem'        => 'Archlinux'
+        }
       }.freeze
 
       DEFAULT_OPERATING_SYSTEMS = [
         'RedHat based Linux',
         'Debian based Linux',
         'Windows',
+        'Archlinux',
```

Comment:
I'd like to leave the default list as is for now, in the near future you will be able to customize this with local config.

Reply:
Well I had to try to sneak it in :D